base_extended_security_grid/models/base.py

```diff
@@ -0,0 +1,17 @@
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, models
+from odoo.osv.expression import AND, normalize_domain
+
+
+class Base(models.AbstractModel):
+
+    _inherit = 'base'
+
+    @api.model
+    def read_grid(self, row_fields, col_field, cell_field, domain=None, range=None):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'range'](https://app.codacy.com/app/numigi/odoo-base-addons/pullRequest?prid=4674190)